packages/story-editor/src/elements/video/playPauseButton.js

```diff
@@ -120,6 +120,7 @@ function PlayPauseButton({
   elementRef,
   element,
   videoRef = null,
+  allowKeyboardSupport = false,
```

Comment:
When would we _not_ want keyboard support?

Reply:
@swissspidy the `playPauseButton` is also being used in trim component. My thought was we dont want user to use spacebar to toggle between play/pause while trimming the video.

Reply:
Actually, it's possible to play/pause while trimming the video using the mouse, so it can be allowed with keyboard as well if the focus is on the play button.

Reply:
By default the focus while trimming would be on `cancel` button if user needs to use the keyboard support then they might have to click on the video element to be able to use the keyboard support if they do that wouldnt it be easier to use the mouse for play/pause button to toggle the video?

Reply:
Sure, but the editor generally also aims to support keyboard-only users. There's no need to specifically forbid keyboard support in trim mode.

Reply:
ok then i guess there wouldnt be any need to have allowKeyboard support prop and also no need to have any separate logic to handleKeydown, will update the code.